dataset/test/bin_test.cpp

```diff
@@ -133,9 +133,11 @@ class BinTest : public ::testing::TestWithParam<DataArray> {
       makeVariable<double>(Dims{Dim::Y}, Shape{3}, Values{-2, -1, 2});
 
   void expect_near(const DataArrayConstView &a, const DataArrayConstView &b) {
-    const auto tolerance = max(buckets::sum(a.data())) * (1e-14 * units::one);
-    EXPECT_TRUE(all(is_approx(buckets::sum(a.data()), buckets::sum(b.data()),
-                              tolerance))
+    const auto tolerance =
+        values(max(buckets::sum(a.data())) * (1e-14 * units::one));
+    EXPECT_TRUE(all(is_close(values(buckets::sum(a.data())),
+                             values(buckets::sum(b.data())), 0.0 * units::one,
+                             tolerance))
```

Comment:
Why is the call to `values` necessary now?

Reply:
Otherwise comparing variances also (causes mismatch) not sure if that's worth investigating as part of this.